cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/09-make-a-plan.html

```diff
@@ -0,0 +1,430 @@
+            {% set expandable_settings = {
+                'label': 'Costs',
+                'value': '$99,999',
+                'is_editable': 'True',
+                'status': 'warning',
+                'data_attribute': 'data-financial-item="total_costs"',
+                'note': '<strong>Minimize indirect costs:</strong> Can you reduce your indirect costs by living at home or buying a refurbished laptop?' | safe
+            } %}
+            {% call() expandable(expandable_settings) %}
+                {{
+                    input.render({
+                        'label': 'Tuition and fees',
+                        'id': 'plan__tuition',
+                        'value': '0',
+                        'data_attribute': 'data-financial-item="dirCost_tuition"'
+                    })
+                }}
+                {{
+                    input.render({
+                        'label': 'Housing and meals',
+                        'id': 'plan__room-board',
+                        'value': '0',
+                        'data_attribute': 'data-financial-item="dirCost_housing"'
+                    })
+                }}
+                {{
+                    input.render({
+                        'label': 'Other direct costs',
+                        'id': 'plan__otherDirectCosts',
+                        'value': '0',
+                        'data_attribute': 'data-financial-item="dirCost_other"'
+                    })
+                }}
+                {{
+                    input.render({
+                        'label': 'Books',
+                        'id': 'plan__books',
+                        'value': '0',
+                        'data_attribute': 'data-financial-item="indiCost_books"'
+                    })
+                }}
+                {{
+                    input.render({
+                        'label': 'Other expenses',
+                        'id': 'plan__other',
+                        'value': '0',
+                        'data_attribute': 'data-financial-item="indiCost_other"'
+                    })
+                }}
+                {{
+                    input.render({
+                        'label': 'Other indirect costs',
+                        'id': 'plan__otherIndirectCosts',
+                        'value': '0',
+                        'data_attribute': 'data-financial-item="indiCost_otherIndirect"'
+                    })
+                }}
+            {% endcall %}
+
+            {% set expandable_settings = {
+                'label': 'Grants & scholarships',
+                'value': '$99,999',
+                'is_editable': 'True',
+                'data_attribute': 'data-financial-item="total_grantsScholarships"',
+                'note': '<strong>Apply for additional grants and scholarships:</strong> Ask the financial aid office and your school counselor (if you’re currently in high school) for help. The Department of Education has more ideas here.' | safe
```

Comment:
Should the "more ideas here" be a link to a DOE resource?

Reply:
Yes, I think so. I should have mentioned in the intro comment that there are some content updates that didn't make it into this PR, so that should be fixed in an upcoming PR.